tools/db/data/protocol-entries.data.ts

```diff
@@ -0,0 +1,170 @@
+import { getDeploymentByName } from './deployments.data';
+import { getOrganizationIdAsStringByName } from './organizations.data';
+import { getUserByUsername } from './test-users';
+import { getUnitByName, getUnitIdAsStringByName } from './units.data';
+
+const getUserAsProducer = (
+	username: Parameters<typeof getUserByUsername>[0],
+): UserProducerDocument => {
+	const user = getUserByUsername(username);
+
+	return {
+		userId: user.id,
+		firstName: user.firstName,
+		lastName: user.lastName,
+		type: ProducerType.USER_PRODUCER,
+	};
+};
+
+const getRescueStationInformation = (
+	rescueStationName: Parameters<typeof getDeploymentByName>[0],
+) => {
+	const rescueStation = getDeploymentByName(rescueStationName);
+	return {
+		rescueStationCallSign: rescueStation.callSign,
+		rescueStationId: rescueStation._id.toString(),
+		rescueStationName: rescueStation.name,
+	};
+};
+
+const getRescueStationMessagePayload = (
+	rescueStationName: Parameters<typeof getRescueStationInformation>[0],
+	alertGroupNames: Parameters<typeof getAlertGroupIdAsStringByName>[0][],
+	unitNames: Parameters<typeof getUnitByName>[0][],
+	strength?: RescueStationMessagePayload['strength'],
+) => {
+	const alertGroups = alertGroupNames.map((alertGroupName) =>
+		getAlertGroupIdAsStringByName(alertGroupName),
+	);
+	const units = unitNames.map((unitName) => getUnitIdAsStringByName(unitName));
+
+	return {
+		...getRescueStationInformation(rescueStationName),
+		strength: strength ?? {
+			leaders: 1,
+			subLeaders: 2,
+			helpers: 3,
+		},
+		alertGroups,
+		units,
+	} satisfies RescueStationMessagePayload;
+};
+
+const collectionData = {
+	collectionName: 'protocol-entries',
+	entries: [
+		{
+			_id: new Types.ObjectId('66846ba41af826ce7c50c3a4'),
+			createdAt: new Date(),
+			updatedAt: new Date(),
+			channel: 'D',
+			payload: { message: 'Test' },
+			producer: getUserAsProducer('testuser'),
+			recipient: {
+				type: UnitType.REGISTERED_UNIT,
+				unitId: getUnitByName('MRB Greif 5')._id,
+			},
+			sender: {
+				type: UnitType.REGISTERED_UNIT,
+				unitId: getUnitByName('ATV')._id,
+			},
+			searchableText: 'Test',
+			time: new Date(),
+			type: ProtocolEntryType.COMMUNICATION_MESSAGE_ENTRY,
+			orgId: getOrganizationIdAsStringByName('Test Organisation'),
+		} satisfies CommunicationMessageDocument,
+		{
+			_id: new Types.ObjectId('6696c4bddfe9fcb08c639c78'),
+			createdAt: new Date(),
+			updatedAt: new Date(),
+			channel: 'D',
+			payload: getRescueStationMessagePayload(
+				'DLRG Einsatzzentrale HH',
+				[],
+				['MRB Greif 5', 'ATV'],
+			),
+			producer: getUserAsProducer('testuser'),
+			recipient: {
+				type: UnitType.REGISTERED_UNIT,
+				unitId: getUnitByName('MRB Greif 5')._id,
+			},
+			sender: {
+				type: UnitType.REGISTERED_UNIT,
+				unitId: getUnitByName('RW Wittenbergen')._id,
+			},
+			searchableText:
+				'Einmeldung - DLRG Einsatzzentrale HH - MRB Greif 5, ATV - Stärke 1/2/3/6',
```

Comment:
can you adjust these texts to fit the result of the `searchableText` to the actual results of the `RescueStationMessageFactory` to avoid confusion?